pinata/test/hello.php

```diff
@@ -0,0 +1,5 @@
+<?php
```

Comment:
Let's remove this file.  
Inside the `s3` folder...there's an image. Please create a new folder called `image` (no plural) and move the image into it. Then you can use that for your test.